### DISCUSSION TITLE
Best practise for rules to set multiple accounts transaction to have the same category

### DISCUSSION THREAD

**Discussion by dmshimself** [firefly-iii/firefly-iii#9118] [Firefly III]
I understand that strict rules cannot use OR conditions and that non-strict rules just any any condition to match.  For me I'd like to set the category for any transaction that matches a whole load of shops I use with one account to use the same category.  A rule which allowed account1 or account2 or account3 would be excellent, but as that isn't yet supported, I have ended up with 3 rules, 1 for each account.

Is there a better way as I'm creating a lot of rules at the moment after importing data from my bank?

**Comment by JC5** [comment]
In the next release of Firefly III, you can set the category to _literally_ (exactly) the destination account, if that is what you're looking for. That would bring it back to one rule.

Otherwise, you may be able to program something yourself using the API and webhooks, but it depends a bit on your own technical skills.

**Comment by dmshimself** [comment]
Thanks for the thoughts.  What I'm looking for is something like Account is A or Account is B or Account is C ...  But I'm sure there are many things on the to do list :-)

**Comment by cardozaj** [comment]
Hey @dmshimself,
I think I did something like what you are wanting to do just a few days ago.
What I ended up doing was going through every expense account that I know is a restaurant/fast food place and I gave those accounts a standardized account number that I came up with.
For example, my expense account for Chick-fil-A has an account number of RES-01-ChiFilA, and all other restaurant/fast food expense accounts have an account number that starts with RES-01-
Then in the rule I set up I set the condition as "Account number starts with" a value of "RES-01-" and have the rule set the budget of the transaction to "Dining Out" 
Hope this helps with what you are wanting to do.

**Comment by dmshimself** [comment]
That sounds a very neat idea.  Many thanks and I'll give that a try
